basics/plugin_structure/plugin_icon_file.md

```diff
@@ -1,42 +1,93 @@
+See [Working with Icons and Images](/reference_guide/work_with_icons_and_images.md) for more information.
 
-**Note:** icons and images that are used within a plugin have different requirements. 
-See [Working with Icons and Images](/reference_guide/work_with_icons_and_images.md)
-for more information.
+* bullet list
+{:toc}
 
-### Plugin Icon File Format
-All Plugin Icon images must be provided in SVG format. 
-Plugin Icons will be displayed in 40px by 40px and 80px by 80px size. However, _only one
-size icon_ should be supplied because it will automatically
-scale.
+## Introduction
+Plugin Logos are shown in the [Plugins Repository](https://plugins.jetbrains.com/), and in [MarketPlace](https://plugins.jetbrains.com/marketplace).
+They also appear in the Settings/Preferences [Plugin Manager](https://www.jetbrains.com/help/idea/managing-plugins.html) UI in JetBrains' IntelliJ Platform-based IDEs. 
+Whether online or in the product UI, a Plugin Logo helps users to identify a plugin more quickly in a list, as shown below:
 
-| Icon Size  | Example SVG Icon                            |
-| ------------- | -------------                            |
-| 40x40         | ![Kotlin Plugin Icon](img/kotlin40.svg)  |
+![Example Product Plugin Preferences Dialog](img/plugin_prefs.png){:width="800px"}
 
-### Plugin Icon File Naming Convention
-Plugin Icon files are named according to the following conventions:
-* `pluginIcon.svg` for use with the default (light) JetBrains IDE theme
-* `pluginIcon_dark.svg` for use with the Darcula JetBrains IDE theme
+## Plugin Logo Icon Requirements
+Plugin Logos are represented as icons, and to be displayed correctly within an IntelliJ Platform-based IDE the icon must:
+* Be the correct file format,
+* Follow the best practices design guidelines
+* Conform to file name conventions.  
 
-### Adding Plugin Icons to a Plugin Project
-The Plugin Icon files must be in the `META-INF` folder of the plugin distribution file, 
-i.e. the `*.jar` or `*.zip` file you upload to the plugin repository and install into
-a JetBrains IDE.
+### Plugin Logo Icon File Format
+All Plugin Logo images must be SVG format. 
+This vector image format is required because the Plugin Logo is displayed in more than one size.
 
-To include Plugin Icons in your distribution file, place the Plugin Icon files 
-into a plugin project's `resources/META-INF` folder. Note this requirement
+### Plugin Logo Size
+The Plugin Logo icon should be provided in one size: 40px x 40px. 
+However, the icons display in two sizes, and scale automatically in each context:
+* 40px by 40px in the plugins _list_ in the Plugin Manager UI.
+* 80px by 80px in the plugin _details screen_ in the Plugin Manager UI, on the plugin's page in the Plugins Repository, and on MarketPlace.
+
+Verify that Plugin Logo icon designs are effective in both sizes and all display contexts.
+
+### Plugin Logo Shape and Visual Weight
+The nominal Plugin Logo size is shown below. 
```

Comment:
Yes, my text was repetitive. Good call.

The spec you sent me states the Plugin Logo size in two places. I'd prefer to only maintain the 40px by 40px size specification in one place. Actually, I'd rather the associated graphic didn't say "36px by 36px" because if the logo spec size changes then I have to change graphics as well as text.

I suggest just shortening the opening sentence to: "Plugin Logo designs should leave at least 2px transparent padding around the perimeter, as shown below."